### ISSUE TITLE
Failed prop type: t: prop type `lang` is invalid; it must be a function, usually from the `prop-types` package, but received `object`.

### ISSUE THREAD

**Issue by abbazer** [jakezatecky/react-checkbox-tree#109]
in [Documentation Properties](https://github.com/jakezatecky/react-checkbox-tree#properties) written that lang have to be an **obj**

my lang obj
```
const lang = {
	collapseAll: "COLLAPSE_ALL",
	expandAll: "EXPAND_ALL",
	toggle: "TOGGLE"
};
```
render()
```
<CheckboxTree
	nodes={categories} checked={cat_checked} expanded={cat_expanded} lang={lang}
	onCheck={cat_checked => this.setState({cat_checked})}
	onExpand={cat_expanded => this.setState({cat_expanded})}
/>
```
**Failed prop type: t: prop type `lang` is invalid; it must be a function, usually from the `prop-types` package, but received `object`.**

What can be the problem?

**Comment by jakezatecky** [comment]
Thank you so much for this issue.

The problem is that I made a mistake and for did not construct the PropType validation for `lang` correctly. I am annoyed that my continuous integration tool does not error out when PropType validations fail.

I have fixed this issue and pushed out v1.3.1.

**Comment by abbazer** [comment]
@jakezatecky cool. thx for this awesome plugin
